packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +70,81 @@ export const Default = {
-                title: 'long with fixed width'
-              }
-            ]
-          }
-        },
-        args: overrideArgs([
-          {
-            type: 'attribute',
-            name: 'open',
-            value: true
-          },
-          {
-            type: 'attribute',
-            name: 'placement',
-            value: 'bottom'
-          }
-        ]),
-        constants: [
-          {
-            type: 'template',
-            name: 'width',
-            value: '<div class="template-position">%TEMPLATE%</div>'
-          }
-        ]
-      })}
-    </div>`;
-  }
+export const Trigger = {
+  render: () => html` <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg" trigger="click"></sd-tooltip> `
 };
 
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['default', 'content'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'slot',
-          name: 'default',
-          title: 'slot=...',
-          values: [
-            {
-              value: `<div class="slot slot--border slot--background slot--text" style="--slot-content: ''; --slot-height: 24px; --slot-width: 24px;"></div>`,
-              title: 'default'
-            },
-            {
-              value: `<div class="slot slot--border slot--background slot--text" slot='content' style="--slot-content: ''; --slot-height: 22px; --slot-width: 82px;"></div>`,
-              title: 'content'
-            }
-          ]
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'width',
-          value: '<div class="template-position">%TEMPLATE%</div>'
-        },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    });
-  }
-};
-
-export const Mouseless = {
-  parameters: {
-    controls: {
-      exclude: ['open']
-    }
-  },
-  render: (args: any) => {
-    return html`<div class="mouseless template-position template-height">${generateTemplate({ args })}</div>`;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-tooltip');
-    await waitUntil(() => el?.shadowRoot?.querySelector('#tooltip'));
-    await waitUntil(() => el?.shadowRoot?.querySelector('sd-icon'));
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    await userEvent.type(el!.shadowRoot!.querySelector('sd-icon')!, '{return}', { pointerEventsCheck: 0 });
-  }
+/**
+ * Use the `hoist` attribute to prevent the tooltip from being clipped when it's placed inside a container with `overflow: auto | hidden | scroll`.
```

Comment:
For the hoist sample: can we use a "bottom-start instead"?
So that when we click the trigger has some margin to the left side.

Reply:
Updated